administration/src/application/components/ApplyController.tsx

```diff
@@ -3,16 +3,19 @@ import '@fontsource/roboto/400.css'
 import '@fontsource/roboto/500.css'
 import '@fontsource/roboto/700.css'
 
-import { useAddEakApplicationMutation, useGetDataPolicyQuery } from '../../generated/graphql'
+import { useAddEakApplicationMutation, useGetDataPolicyQuery, useGetRegionsQuery } from '../../generated/graphql'
 import { DialogActions, Typography } from '@mui/material'
 import useVersionedLocallyStoredState from '../useVersionedLocallyStoredState'
 import { useGarbageCollectArrayBuffers, useInitializeGlobalArrayBuffersManager } from '../globalArrayBuffersManager'
 import ApplicationForm from './forms/ApplicationForm'
-import { useCallback, useMemo, useState } from 'react'
+import { useCallback, useContext, useMemo, useState } from 'react'
 import { SnackbarProvider, useSnackbar } from 'notistack'
 import styled from 'styled-components'
 import DiscardAllInputsButton from './DiscardAllInputsButton'
 import ApplicationErrorBoundary from '../ApplicationErrorBoundary'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import ErrorHandler from '../../ErrorHandler'
+import { Spinner } from '@blueprintjs/core'
```

Comment:
Use  CircularProgress from mui instead. (Also add some margin/padding :) )

Reply:
ok i just centered the spinner